CHANGELOG.md

```diff
@@ -44,6 +44,11 @@
   [Hossam Ghareeb](https://github.com/hossamghareeb)
   [#1763](https://github.com/realm/SwiftLint/issues/1763)
 
+* Fix handling of attributes (`lazy`, `objc`, etc.) for the `let_var_whitespace`
+  rule.
```

Comment:
Hi @Uncommon. Don't forget the two trailing spaces after the description. More info [here](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes)